### ISSUE TITLE
code review

### ISSUE THREAD

**Issue by ruotianliu** [WenjieLuo2333/TGS_SALT#1]
Good job! From the results, you have accomplished your project successfully with your models, because both training set and validation set have achieved the goal of low loss and high accuracy. It is very impressive to combine U-net with Resnet blocks to build your convolutional network and use augmentation to enlarge your datasets cleverly. However, one thing that confuses me is that how do you calculate your final test accuracy? Because your best accuracy is 77%, while according to your diagram of results, your validation accuracy can achieve 95%. There is an obvious gap between them.

**Comment by WenjieLuo2333** [comment]
> Good job! From the results, you have accomplished your project successfully with your models, because both training set and validation set have achieved the goal of low loss and high accuracy. It is very impressive to combine U-net with Resnet blocks to build your convolutional network and use augmentation to enlarge your datasets cleverly. However, one thing that confuses me is that how do you calculate your final test accuracy? Because your best accuracy is 77%, while according to your diagram of results, your validation accuracy can achieve 95%. There is an obvious gap between them.

That's because the Validation Set has only 1600 images. And there are 18000 images for test set. And I use mean IOU to calculate the loss. That is  (prediction∩real_mask)/(prediction∪real_mask)

**Comment by WenjieLuo2333** [comment]
I enlarged the validation set and get a higher accuracy at 79%.
